lib/benefits_intake_service/service.rb

```diff
@@ -24,6 +24,15 @@ class Service < Common::Client::Base
 
     REQUIRED_CREATE_HEADERS = %w[X-VA-First-Name X-VA-Last-Name X-VA-SSN X-VA-Birth-Date].freeze
 
+    # Validate a file satisfies Benefits Intake specifications. File must be a PDF.
```

Comment:
@Thrillberg has been working on a metadata validator that could be beneficial here as well

Reply:
Thanks @tblackwe ! Indeed, I have [a metadata validator](https://github.com/department-of-veterans-affairs/vets-api/blob/3f0c17d7eab9b162cfaa5daa6bdbb26b926941f9/lib/simple_forms_api_submission/metadata_validator.rb#L4) that will either adjust your metadata to conform to the Benefit Intake API's validation rules or raise an exception if it's unable to.

If you want to use it you might be able to do something like:
```
metadata = SimpleFormsApiSubmission::MetadataValidator.validate(generate_metadata(form_metadata))
```
on line 49 below. Let me know if you are interested or have any questions about it!

Reply:
acknowledged and approved